### ISSUE TITLE
Refresh Stock Prices from Yahoo error "url using bad/illegal format or missing url" #6040

### ISSUE THREAD

**Issue by swjpanton** [moneymanagerex/moneymanagerex#6049]
Hello Why is this issue kept being closed when i don't understand what is going on

 - [ ] Mac OSX
 - [ ] i keep getting this errors and the reply i get is case closed and referred to a load of messages, none of which i understand.

Please may I have a response that I can actually understand and if there is something i need to do please can it be explained in laymans terms as i am not a technical programmer.
![screenshot 1](https://github.com/moneymanagerex/moneymanagerex/assets/107315794/3aea4f49-05be-46c9-97b3-9dd8afc0c510)

**Comment by tactilis** [comment]
@swjpanton
Please see my comment at the end of #6048.

This is the **_3rd_** duplicate you have raised.

**_Please stop raising duplicates._** You are making extra work for people.

All duplicates will be closed and pointed to the source issue #6040, which is still open.

Please read that issue.

The bug, which affects only some MMEX installations, is not fixed.

**Comment by swjpanton** [comment]
The source does not discuss the operating system I use so its not duplicate and nor understandable to a lay personSent from my iPhoneOn 24 Jun 2023, at 19:24, Martin ***@***.***> wrote:﻿
@swjpanton
Please see my comment at the end of #6048.
This is the 3rd duplicate you have raised.
Please stop raising duplicates. You are making extra work for people.
All duplicates will be closed and pointed to the source issue #6040, which is still open.
Please read that issue.
The bug, which affects only some MMEX installations, is not fixed.

**Comment by tactilis** [comment]
> The source does not discuss the operating system I use so its not duplicate and nor understandable to a lay person

Please see https://github.com/moneymanagerex/moneymanagerex/issues/6048#issuecomment-1605733713

**Comment by tactilis** [comment]
This duplicate issue can be closed now in favour of #6040.

**Comment by vomikan** [comment]
https://github.com/moneymanagerex/moneymanagerex/issues/6203
